i18n-helpers/src/lib.rs

```diff
@@ -1666,4 +1689,27 @@ after-no-comment
             ]
         );
     }
+
+    #[test]
+    fn extract_message_translator_comments_on_codeblock() {
+        assert_eq!(
+            extract_messages(
+                r#"
+<!-- mdbook-xgettext:comment: greetings! -->
+```python
+print("Hello world")
+```
+"#
+            )
+            .into_iter()
```

Comment:
Is `extract_messages` not already returning a `Vec`?

Reply:
Whooops; thank you.  This must have been some residue from earlier work.  Dropped.